meowlflow/build.py

```diff
@@ -72,7 +72,7 @@
 EXPOSE 8000
```

Comment:
and remove this?

Reply:
Yes, this doesn't actually do anything, it's just informative